frontend/public/locales/en/common.json

```diff
@@ -203,5 +203,7 @@
   "cancel": "Cancel",
   "delete": "Delete",
   "go-to-page": "Go to {{page}}",
-  "account-deletion-token-mismatch": "The account deletion request failed to authenticate."
-}
+  "account-deletion-token-mismatch": "The account deletion request failed to authenticate.",
+  "privacy-policy": "Privacy Policy",
+  "privacy-policy-description": "By using this website, you agree to the Privacy Policy."
```

Comment:
Maybe you should link here to the Privacy Police

Reply:
The text is used for embeds on other websites. So basically the detail text you get on a twitter or facebook embed. It can probably be improved, but I don't think we can have a link.